packages/app/src/systems/Unlock/hooks/useUnlockForm.tsx

```diff
@@ -12,7 +12,7 @@ const schema = yup
 export type UseUnlockFormReturn = ReturnType<typeof useUnlockForm>;
 
 export type UnlockFormValues = {
-  password: string;
+  password: string | undefined;
```

Comment:
```suggestion
  password?: string;
```

Reply:
Why does this need to accept undefined?

Reply:
because now for props inside of schema object that are not `required`,  `yupResolver` will accept only together with `undefined` typing